manual/CHAPTER_CellLib.tex

```diff
@@ -273,6 +274,15 @@ \subsection{Memories}
 The cell with the higher integer value in this parameter wins a write conflict.
 \end{itemize}
 
+The {\tt \$meminit} cells have an address input \B{ADDR} and a data input \B{DATA}. Both of the inputs
+must resolve to a constant for synthesis to succeed. If two {\tt \$meminit} cells have the same
+\B{MEMID} parameter and \B{ADDR} input, the contents of that memory at that address is undefined.
```

Comment:
Just documenting the current behavior of `memory_collect`. It would be prudent to check for duplicate addresses, I think, if this is not explictily desired to make this undefined.

Reply:
Actually, now that I look closer there is a `PRIORITY` parameter... that promptly gets ignored by `memory_collect`.

Reply:
No, `memory_collect` uses `PRIORITY`. See `memcells_cmp()`. The `$meminit` cell with higher priority overwrites the values from cells with lower priority.